uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/FeaturesServerTest.java

```diff
@@ -110,7 +110,7 @@ private void downloadRuntime(String name) {
 		runtimeWiz.setInstallFolder(getPath() + "target/temp_install");
 		runtimeWiz.setDownloadFolder(getPath() + "target/temp");
 		runtimeWiz.finish(name);
-		new WaitWhile(new ShellIsAvailable("Download 'JBoss Fuse 6.3.0'"), TimePeriod.getCustom(900));
+		new WaitWhile(new ShellIsAvailable("Download 'Red Hat Fuse 6.3.0'"), TimePeriod.getCustom(900));
```

Comment:
need to be kept as-is for 6.x https://issues.jboss.org/browse/FUSETOOLS-2843?focusedCommentId=13545002&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13545002